frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewSidePanel/FeatureOverviewSidePanelDetails/DependencyRow.tsx

```diff
@@ -0,0 +1,96 @@
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { Button } from '@mui/material';
+import { Add } from '@mui/icons-material';
+import { TooltipLink } from 'component/common/TooltipLink/TooltipLink';
+import { AddDependencyDialogue } from 'component/feature/Dependencies/AddDependencyDialogue';
+import { useUiFlag } from 'hooks/useUiFlag';
+import { IFeatureToggle } from 'interfaces/featureToggle';
+import { FC, useState } from 'react';
+import { FlexRow, StyledDetail, StyledLabel, StyledLink } from './StyledRow';
+
+export const DependencyRow: FC<{ feature: IFeatureToggle }> = ({ feature }) => {
```

Comment:
moved this code from a previous location so that everything related to dependencies can be managed in one file